ovos_PHAL_plugin_balena_wifi/__init__.py

```diff
@@ -5,9 +5,9 @@
 
 import pexpect
 from ovos_bus_client.message import Message, dig_for_message
+from ovos_bus_client.apis.gui import GUIInterface
```

Comment:
should require minimum version of bus client that introduced this import in requirements.txt